src/matrix/inversewishart.jl

```diff
@@ -56,6 +56,7 @@ insupport(d::InverseWishart, X::Matrix) = size(X) == size(d) && isposdef(X)
 dim(d::InverseWishart) = dim(d.Ψ)
 size(d::InverseWishart) = (p = dim(d); (p, p))
 size(d::InverseWishart, i) = size(d)[i]
+rank(d::InverseWishart) = dim(d)
```

Comment:
Unless `rank` has been `import`ed, this will need a module qualification in order to extend the binding from LinearAlgebra rather than defining a new local binding.

Reply:
It's imported:

https://github.com/JuliaStats/Distributions.jl/blob/91b9df8ab88fb08aabc4294b7dfcbb2ee1c7ae07/src/Distributions.jl#L12

Reply:
Okay perfect.